app/controllers/idv/document_capture_controller.rb

```diff
@@ -96,18 +92,14 @@ def cancel_establishing_in_person_enrollments
     end
 
     def analytics_arguments
-      liveness_checking_required =
-        FeatureManagement.idv_allow_selfie_check? &&
-        resolved_authn_context_result.biometric_comparison?
```

Comment:
Per @aduth's excellent earlier comments, I removed these variable assignments that were now `x = y` and just inlined the `y` part where it was used. This plays out over the next handful of blocks.